### ISSUE TITLE
Sending `onInfoUpdate` with no listeners registered.

### ISSUE THREAD

**Issue by megacherry** [adaptyteam/AdaptySDK-React-Native#18]
hi,

i've get a warning in iOS, but only if i start the app. I get the warning "Sending `onInfoUpdate` with no listeners registered."

my code looks like:

```
useEffect(() => {
    const loadAdapty = async () => {
      await activateAdapty({
        sdkKey: Config.ADAPTY_PUBLIC_SDK_KEY,
        customerUserId: userId,
      });
    };

    loadAdapty() && setInitAdapty(true);
  }, []);
```

thanks in advance.

best regards
Andy

**Comment by Betra** [comment]
Hey! There's a way to disable this, give me a few hours to fix this!

**Comment by Betra** [comment]
Thanks for your concern!

**Comment by divanc** [comment]
Fixed since `1.3.11`
